### DISCUSSION TITLE
mox script workflow?

### DISCUSSION THREAD

**Discussion by jdduprey** [sr320/course-fish546-2021#105] [Q&A] [answered]
I may have missed this in the tutorial materials, but how do you typically organize the slurm scripts you want to run on mox? I used `scp` to upload the script to a scripts folder in my scrubbed directory, then ran `sbatch mox-blast.sh`. Is there a better way to do this?

**Comment by kubu4** [comment, marked as answer]
It's up to you. 

My preference is to house the SLURM script in the working directory that the script is operating upon. That way, when a job finishes and I `rsync` the working directory to Gannet, the script used to run the job is included and is then available for reference in the future.

**Reply by jdduprey** [reply]
Thanks Sam, that makes a lot of sense. Follow up question: How long would you expect it would take to run the example blast job?

**Reply by kubu4** [reply]
No real way to answer that. All depends on how "big" your data is and how "big" your database is, as well as CPU settings, etc. If you're wondering about what to set for a time in your SLURM script, you can set it for 7 days. Currently, that's the max, as Mox will be down for maintenance on 2/9/2021.
